components/CommentCard/CommentCard.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useRef, useState } from "react";
+import styled from "styled-components";
+import ChevronDown from "../../public/icons/chevron_down";
+import ChevronUp from "../../public/icons/chevron_up";
+
+export default function ThePage({ comment, time }) {
+  const descRef = useRef();
```

Comment:
I don't understand the abbreviation, wouldn't it be better (for future Jin as well) to be more precise here?